packages/contracts/src/abis/foxyStaking.ts

```diff
@@ -1,6 +1,4 @@
-import type { InterfaceAbi } from 'ethers'
-
-export const FOXY_STAKING_ABI: InterfaceAbi = [
+export const FOXY_STAKING_ABI = [
```

Comment:
Can we remove from investor-foxy as well and  leverage this guy? 

https://github.com/shapeshift/web/blob/72aed01fb9be296e39fa350e8d195d3d1b6d12b5/src/lib/investor/investor-foxy/abi/foxy-staking-abi.ts#L3

Reply:
Absolutely, and the rest 

![image](https://github.com/user-attachments/assets/272e53ba-e9b6-4e64-9b0f-bda394dddb22)

Reply:
https://github.com/shapeshift/web/pull/7687/commits/18a308d6018a6d80390eee6e66a7afbeade0880e#diff-853b847b7d44b7f4096332e961bd39305496f7ef8ee2d29bd64c448a4171e4f2 looks perf!